vscode/src/organization.ts

```diff
@@ -23,11 +23,15 @@ async function updateStatus(ctx: Context) {
 }
 
 async function executeCommand(ctx: Context) {
-  await ensureAuth(ctx);
-  const org = await selectOrganization(ctx);
-  await setOrganization(ctx, org);
-  await updateStatus(ctx);
-  ctx.refreshDeviceView();
+  try {
```

Comment:
Seems unrelated.

Reply:
Must have forgotten to use the correct base.